src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+AnnotationAssertion(rdfs:label obo:CL_4033030 "diffuse bipolar 3b cell")
+SubClassOf(obo:CL_4033030 obo:CL_0000750)
+
+# Class: obo:CL_4033031 (diffuse bipolar 4 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27833534") Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033031 "An ON diffuse bipolar cell that has preferential connectivity with ON parasol cells and bistratified amacrine cells.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033031 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033031 "2023-04-04T09:51:10Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033031 "DB4 cell")
+AnnotationAssertion(rdfs:label obo:CL_4033031 "diffuse bipolar 4 cell")
+SubClassOf(obo:CL_4033031 obo:CL_0000749)
+
+# Class: obo:CL_4033032 (diffuse bipolar 6 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033032 "An ON diffuse bipolar cell that has large axon terminals which show little or no overlap. This cell has a preferential connectivity to narrow thorny ganglion cells.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033032 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033032 "2023-04-04T09:51:22Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033032 "DB6 cell")
+AnnotationAssertion(rdfs:label obo:CL_4033032 "diffuse bipolar 6 cell")
+SubClassOf(obo:CL_4033032 obo:CL_0000749)
+
+# Class: obo:CL_4033033 (flat midget bipolar cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033033 "An OFF bipolar cell with a small dendritic tree that provides most of the triad-associated basal (flat) contacts at cone pedicles.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033033 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033033 "2023-04-04T10:54:00Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033033 "FMB cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033033 "A flat midget bipolar cell expresses the calcium binding protein recoverin.")
```

Comment:
... noted in macaque and human, but not marmoset retinas.

Reply:
I have removed the comment, thank you.